src/scripts/main.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('table');
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+
+const tableRows = table.rows;
+
+const tbodyRows = [...tbody.rows];
+
+const headRow = tableRows[0];
+const headElements = [...headRow.children];
+
+thead.addEventListener('click', (e) => {
+  const link = e.target.closest('th');
+
+  if (!link) {
+    return;
+  }
+
+  const index = headElements.findIndex(
+    (el) => el.textContent === link.textContent,
+  );
+
+  const sortType = () => {
+    switch (link.textContent) {
+      case 'Age':
+        return 'number';
+
+      case 'Salary':
+        return 'salary';
+
+      case 'Name' || 'Position':
```

Comment:
The condition `case 'Name' || 'Position':` is incorrect. The `||` operator does not work as expected in a switch case. You should use separate cases for 'Name' and 'Position'.